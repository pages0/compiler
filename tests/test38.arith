let x : <(int*int*bool*(int*()))> = (ref ( 3,5, false,(5,())))
in
!x