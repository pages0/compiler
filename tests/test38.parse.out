( let [ x ( ref ( tuple 3 5 false ( tuple 5 () ) ) ) ] ( ! x ) )